docs/tutorials/plots.rst

```diff
@@ -0,0 +1,966 @@
+:orphan:
+
+.. include:: ../references.txt
+
+.. _plots:
+
+.. doctest-skip-all
+
+***********************
+Plotting with Astroplan
+***********************
+
+`astroplan` currently has convenience functions for making three different types
+of plots: airmass vs time, parallactic angle vs time and sky charts.  While
+`astroplan` requires `Matplotlib`_, the use of additional plotting packages
+(such as `Seaborn`) is neither explicitly prohibited or supported.
+
+All `astroplan` plots return a `matplotlib.axes.Axes` object (often referred
+to as `ax`).  You can further manipulate the returned `ax` object (including
+using it as input for an `astroplan` plotting function, see examples below)
+or you can simply display/print the plot.
+
+Contents
+========
+
+    * :ref:`plots_time_dependent`
+
+    * :ref:`plots_sky_charts`
+
+.. warning::
+
+    All examples here assume you know how to and have already constructed
+    `astroplan.Observer` and `astroplan.FixedTarget` objects.
+
+
+.. _plots_time_dependent:
+
+Time Dependent Plots
+====================
+
+Although all `astroplan` plots are time-dependent in some way, we label those
+that have a time-based axis as "time-dependent".
+
+`astroplan` currently has two different types of "time-dependent" plots:
+
+* Airmass vs. Time
+* Parallactic Angle vs. Time
+
+These take, at minimum, `astroplan.Observer`, `astroplan.FixedTarget` and
+`astropy.time.Time` objects as input.
+
+.. _plots_airmass:
+
+Airmass versus time plots are made in the following manner::
+
+    plot_airmass(target, observer, time)
+
+.. _plots_parallactic:
+
+Parallactic angle vs time plots are made in the following manner::
+
+    plot_parallactic(target, observer, time)
+
+Below are general guidelines for working with time-dependent plots in
+`astroplan`.  Examples use airmass but apply to parallactic angle as well.
+
+.. seealso::
+
+    ???? `astroplan.Observer.altaz.secz` ?????
```

Comment:
ah, this is where I wasn't sure where `secz` was coming from, and wanted to point users to that place--wherever it was!  I've since figured out that it comes from [here](http://astropy.readthedocs.org/en/latest/api/astropy.coordinates.AltAz.html#astropy.coordinates.AltAz.secz).